packages/amplication-git-utils/src/git/github.service.ts

```diff
@@ -3,31 +3,38 @@ import { ConfigService } from "@nestjs/config";
 import { createAppAuth } from "@octokit/auth-app";
```

Comment:
`GithubService` needs to have its own tests to cover business logic declared here.

Reply:
Yes. We should add this test, but I think we can do it after the last step of the refactoring

Reply:
Sure, let's create an issue or a task on the main issue to avoid forgetting this please.